<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Choix</class>
 <widget class="QWidget" name="Choix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>731</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#547a64;color:white;border:0px;padding-left:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; border:0px;</string>
   </property>
   <column>
    <property name="text">
     <string>Validée</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Université</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Formation</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Contacte</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Selectivité</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tarifs-Capacité</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Motivation</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>241</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>241</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteUniv">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:white; color : #17afd3; border: 0px solid #000028 ;  text-align:left; padding-left:5px;}

QPushButton:hover{background:#17afd3;color:white;border: 0px solid #ecf0f5}

QPushButton:focus{background-color:#17afd3;color:white};</string>
   </property>
   <property name="text">
    <string>Supprimer l'université</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../PROCEDURE_UI/ICON/delete.png</normaloff>../../../../PROCEDURE_UI/ICON/delete.png</iconset>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveUniv">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:white; color : #17afd3; border: 0px solid #000028 ;  text-align:left; padding-left:5px;}

QPushButton:hover{background:#17afd3;color:white;border: 0px solid #ecf0f5}

QPushButton:focus{background-color:#17afd3;color:white};</string>
   </property>
   <property name="text">
    <string>Enregistrer le document</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../PROCEDURE_UI/ICON/word.png</normaloff>../../../../PROCEDURE_UI/ICON/word.png</iconset>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUniv">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#17afd3; color : white; border: 0px solid #000028 ;  text-align:left; padding-left:5px;}

QPushButton:hover{background:white;color:#17afd3;border: 0px solid #ecf0f5}

QPushButton:focus{background-color:white;color:white};</string>
   </property>
   <property name="text">
    <string>Ajouter l'université à mes choix</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../PROCEDURE_UI/ICON/addBlack.png</normaloff>../../../../PROCEDURE_UI/ICON/addBlack.png</iconset>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
